packages/components/organisms/f-checkout/src/services/checkoutApi.js

```diff
@@ -0,0 +1,48 @@
+import axios from 'axios';
+import { VUEX_CHECKOUT_EXPERIMENTATION_MODULE } from '../constants';
+
+export default {
+    async getCheckout (url, state, timeout) {
+        const authHeader = state.authToken && `Bearer ${state.authToken}`;
+        const config = {
+            headers: {
+                'Content-Type': 'application/json',
+                ...(state.authToken && {
+                    Authorization: authHeader
+                })
+            },
+            timeout
+        };
+
+        return axios.get(url, config);
+    },
+
+
+    async updateCheckout (url, state, rootGetters, data, timeout) {
+        const authHeader = state.authToken && `Bearer ${state.authToken}`;
```

Comment:
This line is repeated allot, could it be extracted to a method?
```javascript
function generateAuthHeader(authHeader) {
    return authHeader ? `Bearer: ${authHeader}` : '';
}
```

Reply:
Whereas it's a valid comment, I believe this is going to be resolved when we implement what Damian suggested, to remove duplication in all the calls, basically.

Reply:
yes this will be resolved in a follow up ticket in the checkout team :)